src/vignette/storage/static_assets.clj

```diff
@@ -24,7 +24,7 @@
 
 (defn create-async-response-stored-object [static-image-get image-review-get]
   (let [static-image-response @static-image-get]
-    (if (and (-> static-image-response :status (= 200)) (some #(= % (:status @image-review-get)) [200 404]))
+    (if (and (-> static-image-response :status (= 200)) (some #(not (= % (:status @image-review-get))) [403]))
```

Comment:
It seems that I changed it too quickly, allowing vignette to display an image in more than required response statuses from image-review. I've changed it once more.